### PR TITLE
S1task2 display db

### PR DESCRIPTION
**Pull request by Albertoroldanq**
code review please :D

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-// calling a function that belongs to the object
-// changing the default "fetch mode" of the PDO - this affects the format of the data that gets pulled out of the db
-//e.g. PDO::FETCH_ASSOC formats the output from the db as an associative array
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+<html>
+<head>
+    <!--CUSTOM STYLESHEET-->
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
```
Comment (SamBellDev) [inline index.php:11]: The page doesn't display the foreign characters. I would add a meta tag with UTF 8 to better display the foreign characters.

File: functions.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                            <h3>' . $wine['origin'] . '</h3>
+                            <h3>' . $wine['grape'] . '</h3>
+                        </div>
+                      </div>';
+        }
+    return $htmlWineCard;
+}
+
+function addNewWine() {
+}
```
Comment (SamBellDev) [inline functions.php:21]: What is the purpose of addNewWine()?

**Review by SamBellDev — CHANGES_REQUESTED** [review]
Looks clean apart from the char set missing and a rogue empty function.

File: functions.php (hunk 2 above)
Comment (Albertoroldanq) [reply functions.php]: deleted

File: index.php (hunk 1 above)
Comment (Albertoroldanq) [reply index.php]: added

**Pull request closed without merging**